wagtail/blocks/field_block.py

```diff
@@ -708,6 +720,8 @@ def value_from_form(self, value):
 
     def get_searchable_content(self, value):
         # Strip HTML tags to prevent search backend from indexing them
```

Comment:
Just a nitpick here: this comment is describing what the `get_text_for_indexing` function is doing, so it would be best to move this down to just before the `return [get_text_for_indexing(source)]` line.